testsuite/MDAnalysisTests/data/tprs/virtual_sites/dummy.top

```diff
@@ -0,0 +1,161 @@
+; This topology describes a system with a test molecule that intends to
+; use all the bonded interactions available in gromacs.
+; Most of the parameter values make no sense; here only the presence of the
+; interaction in the final TPR matters.
+
+[ defaults ]
+1 1
+
+[ atomtypes ]
+; name mass charge ptype c6          c12
+TA     72.0 0.000  A     0.24145E-00 0.26027E-02
+
+[ moleculetype ]
+; molname  	nrexcl
+TEST 	    	1
+
+[ atoms ]
+;id 	type 	resnr 	residu 	atom 	cgnr 	charge   mass
+1       TA      1       T       T1      1       0
```

Comment:
I am surprised that you didn't name the atoms with emojis...

Reply:
It i a missed opportunity! Though, emojis are not officially supported by gromacs, so maybe they are a bit too hardcore for the tests :stuck_out_tongue: